utils/spacewalk-common-channels

```diff
@@ -169,8 +190,17 @@ Create everything as well as unlimited activation key for every channel:
         print parser.print_help()
         parser.exit()
 
-    key = None
-    client = None
+    try:
```

Comment:
I don't really think it's bad to move connection to here, instead of having it below. That way we can start using it at `add_channels`